mcdemoaux/vision/vis.py

```diff
@@ -40,7 +40,11 @@ def start(self):
         pass
 
     def run(self):
-        pass
+        if len(self.queue) >= 3:
```

Comment:
It's a good approach to drop unnecessary frames. Dropping more frames and setting `waitKey` delay lower works better for me. Dropping `8` instead of `3` frames with `20` ms delay instead of `50` works much better.

I'm also not sure if it makes sense, but `imshow` on each popped image seems excessive. We can send just last image to `imshow`. Not sure if `imshow` does anything before `waitKey` is executed, but jic it's better not to call it, when not needed.
```
data = self.queue.pop()
cv2.imshow(data[0], data[1])
```

Reply:
Ok. WIll change that

Reply:
I've updated the code and tested it. Seems working for me.